enterprise-suite/tests/resources/ingress-test.tls.yaml

```diff
@@ -2,7 +2,6 @@ apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
   name: ingress-test-tls
-  namespace: lightbend
```

Comment:
This still works without this?

Reply:
Yes because we specify the namespace when applying.